recipes/vectorclass/all/CMakeLists.txt

```
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
+
+project(cmake_wrapper)
```

```suggestion
project(vectorclass)
```

not really a wrapper